package.json

```diff
@@ -2,17 +2,22 @@
   "name": "wdio-cucumberjs-json-reporter",
   "version": "2.0.4",
```

Comment:
Hi @yon-cuadrado 

Thanks for the PR and all the work you've put into it. I just want to check-check-double-check before we merge this. The current version is version 3.0.0, and the package.json is at least at `2.0.4`, which one was your main branch? Are you sure all latest changed are in this one?
If so, can you upgrade it to 3.0.0 so the package.json file is also in line with the main branch?

Thans